components/subViews/company/matchList.vue

```diff
@@ -1,36 +1,61 @@
 <template>
     <div class="match-list">
-          <h3>
-            {{ post.title }}
-            {{ $t("looking-for") }}
-          </h3>
-          <div class="match-items">
-            <div class="match-list-items">
-              <ul>
-                <p style="font-weight: 700">{{ $t("programs") }}</p>
-                <li v-for="program in post.programs_data[locale]" :key="program.id">
-                  {{ program }}
-                </li>
-              </ul>
-              <ul>
-                <p style="font-weight: 700">{{ $t("positions") }}</p>
-                <li v-for="position in post.positions_data[locale]" :key="position.id">
-                  {{ position }}
-                </li>
-              </ul>
+      <h3>
+        {{ post.title }}
+        {{ $t("looking-for") }}
+      </h3>
+      <div class="items">
+
+        <!-- Section: Bachelors -->
+        <div class="section">
+          <p class="section-heading">{{ $t("programs") }}</p>
+          <div class="tag-list">
```

Comment:
shouldn't this be a `<ul>` with `<li>` inside?

Reply:
maybe the tag-list. Is that very important tho? What difference does this make?

Reply:
Exactly. It's just HTML best practice, nothing major - but better for accessibility, clarity etc.

Reply:
ok will have a look